src/components/ebay-section-title/examples/01-title/template.marko

```diff
@@ -0,0 +1,3 @@
+<ebay-section-title>
```

Comment:
I would actually think this is a bit too verbose for the default example. 
We need to check if section-title-title render body is there and then use that, otherwise use the default. I did something similar on `ebay-notice` with CTA button. 
I would prefer to do:
```
<ebay-section-title>
   Section Title
</ebay-section-title>
```

I think the nested one should only be used when there's other variations there. 
Or we can always have the title be on it's own and the other variations inside of tags (although I'm fine keeping it the way you have it for the other cases).

Reply:
I updated component so this now works:
```
<ebay-section-title>Section Title</ebay-section-title>
```